tests/test_contacts.py

```diff
@@ -13,6 +13,8 @@
 from sid.contacts import calculate_infections_by_contacts
 from sid.contacts import create_group_indexer
 
+SEED = itertools.count(384)
```

Comment:
I would prefer having `itertools.count()` in every test. Otherwise reproducibility depends on the order by which tests are executed which constantly changes with pytest-xdist and if we change tests.